phoenix-core/src/it/java/org/apache/phoenix/end2end/index/DropColumnIT.java

```diff
@@ -373,9 +373,9 @@ public void testDroppingIndexedColDropsIndex() throws Exception {
             
             // verify that the regular index physical table was dropped
             try {
-                conn.unwrap(PhoenixConnection.class).getQueryServices().getTableDescriptor(indexTablePhysicalName);
+                conn.unwrap(PhoenixConnection.class).getQueryServices().getAdmin().getTableDescriptor(indexTablePhysicalName);
                 fail("Index table should have been dropped");
-            } catch (TableNotFoundException e) {
+            } catch (org.apache.hadoop.hbase.TableNotFoundException e) {
```

Comment:
At this point the problem arose due to table name is being cached in tableCache variable at HBase level. Could not find a way to invalidate it, hence the RPC.  any suggestion?

Reply:
There's no problem with a single RPC in a test like here, just in code that gets called every commit like previously in MutationState below :-)